master/docs/developer/rest.rst

```diff
@@ -112,7 +112,7 @@ Filters can use any of the operators listed below, with query parameters of the
 ``ge``
     select resources where the field's value is greater than or equal to ``{value}``
 ``contains``
-    select resources where the field's value contains ``{value}``
+    select resources where the field's value contains ``{value}``, or with the same parameter appearing multiple times contains one of the values (so `foo__contains=x&foo__contains=y` would match resources where foo contains `x`, `y` or both)
```

Comment:
This is an awkwardly long sentence. The addition in this PR should be its own sentence.